.circleci/config.yml

```
@@ -103,74 +103,81 @@ workflows:
                 - main
           requires:
             - deploy_dev
+build-test-and-deploy:
+  - e2e_environment_test:
+      environment: 'dev'
+      context: hmpps-common-vars
+      filters:
+        branches:
+          only:
+            - 740-booking-extension-contract-test
```

This should look like this:

```suggestion
build-test-and-deploy:
  jobs:
    - e2e_environment_test:
        environment: 'dev'
        context: hmpps-common-vars
        filters:
          branches:
            only:
            - 740-booking-extension-contract-test
```